pkg/jaeger/jaeger_event_test.go

```diff
@@ -0,0 +1,96 @@
+package jaeger
+
+import (
+	"strconv"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+
+	jaegerModel "github.com/jaegertracing/jaeger/model"
+
+	"github.com/pavolloffay/jaeger-local-storage-poc/pkg/model"
+)
+
+func TestJaegerSpan(t *testing.T) {
+	span := &jaegerModel.Span{
+		TraceID:       jaegerModel.NewTraceID(0, 123),
+		SpanID:        jaegerModel.NewSpanID(567),
+		OperationName: "op1",
+		StartTime:     time.Unix(0, 1000),
+		Duration:      5000,
+		Tags: jaegerModel.KeyValues{
+			jaegerModel.String("key1", "value1"),
+			jaegerModel.Bool("key2", true),
+		},
+		Process: &jaegerModel.Process{
+			ServiceName: "serviceA",
+			Tags: jaegerModel.KeyValues{
+				jaegerModel.String("tenant", "tenant1"),
```

Comment:
Could you add a test where the same tag exists at both span and process levels? Same with a tag named `service`, `spanId`, `traceId`, `operation`, and `duration`.

Reply:
Will do - although for the named fields I will differentiate them.

Reply:
If the field names are qualified, no need to compare all the fields then. A cross-test would probably be sufficient.